pyteal/compiler/optimizer/optimizer.py

```diff
@@ -21,13 +22,48 @@ class OptimizeOptions:
     Args:
 
         scratch_slots (optional): cancel contiguous store/load operations
-            that have no load dependencies elsewhere.
+            that have no load dependencies elsewhere. Starting TEAL version 9, defaults to optimizing.
+        frame_pointers (optional): employ frame pointers instead of scratch slots during compilation.
+            Available only starting in version 8. Defaults to optimizing starting TEAL version 8.
```

Comment:
@tzaffi I'm only seeking to comment on a minor verbiage concern.  We've moved towards referencing _program versions_/_AVM versions_ rather than _TEAL versions_.

```suggestion
            that have no load dependencies elsewhere. Starting with program version 9, defaults to optimizing.
        frame_pointers (optional): employ frame pointers instead of scratch slots during compilation.
            Available only starting in program version 8. Defaults to optimizing starting in program version 8.
```